cg/meta/workflow/mip.py

```diff
@@ -74,7 +80,7 @@ def get_priority(self, family_obj: models.Family) -> str:
             return "high"
         return "normal"
 
-    def pedigree_config(self, family_obj: models.Family, pipeline: str) -> dict:
+    def pedigree_config(self, family_obj: models.Family, pipeline: str, **kwargs) -> dict:
```

Comment:
I think it would be more clear and maintainable if the arg is more specific in the call. kwargs is so generic it hides some of the players in the call. "panel_bed" as an args with default None is more readable and specific.